t/ui/PhantomTest.pm

```diff
@@ -110,4 +111,8 @@ sub kill_phantom() {
     waitpid($phantompid, 0);
 }
 
+sub get_mojoport {
+    return $mojoport;
+}
+
```

Comment:
yeah, I don't think you need to export this when just using the method "get_current_url" on the test driver after startup. It will give you the correct URL. I also used it, e.g. see here https://github.com/os-autoinst/openQA/blob/master/t/ui/15-comments.t#L46